srcpkgs/acl/template

```diff
@@ -21,8 +21,7 @@ if [ "$XBPS_TARGET_MACHINE" = "i686" ]; then
 	configure_args="--libdir=/usr/lib32 --libexecdir=/usr/lib32"
 fi
 
-do_install() {
-	make DIST_ROOT=${DESTDIR} install install-lib install-dev
+post_install() {
 	rm -rf ${DESTDIR}/usr/share/doc
```

Comment:
I'd suggest to instead `vmove usr/share/doc` in the `acl-devel` subpackage.
Other than that LGTM.